### PR TITLE
[READY_TO_REVIEW]feature/#377 Added fix new table displayed correctly

### LINKED ISSUE #377: New table is not displayed correctly according to zoom
If you add a table and you have used the zoom buttons the positioning of that new table is not added in the upper left corner of the screen.

### PR DESCRIPTION
**Pull request by Ivanruii**
closed #377

### CODE REVIEW

File: src/pods/toolbar/components/add-collection/set-off-set-zoom-to-coords.hook.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { useCanvasViewSettingsContext } from '@/core/providers/canvas-view-settings';
+import { CANVAS_MAX_HEIGHT, CANVAS_MAX_WIDTH } from '@/core/providers';
+import { Coords } from '@/core/model';
+
+export const useOffsetZoomToCoords = ({ x, y }: Coords): Coords => {
```
Comment (brauliodiez) [inline src/pods/toolbar/components/add-collection/set-off-set-zoom-to-coords.hook.tsx:5]: Missing specs (unit tests?)?

Comment (Ivanruii) [reply src/pods/toolbar/components/add-collection/set-off-set-zoom-to-coords.hook.tsx:5]: I have not done unit test yet.

File: src/pods/toolbar/components/add-collection/set-off-set-zoom-to-coords.hook.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { useCanvasViewSettingsContext } from '@/core/providers/canvas-view-settings';
```
Comment (brauliodiez) [inline src/pods/toolbar/components/add-collection/set-off-set-zoom-to-coords.hook.tsx:1]: The PR contains merge conflicts could you fix it? Thanks

Comment (Ivanruii) [reply src/pods/toolbar/components/add-collection/set-off-set-zoom-to-coords.hook.tsx:1]: Fixed conflicts, ready to merge!

**Pull request merged**